lib/stack_master/commands/apply.rb

```diff
@@ -127,7 +127,13 @@ def update_stack
 
       def upload_files
         return unless use_s3?
-        s3.upload_files(s3_options)
+
+        # Directly use remote templates from S3 bucket
```

Comment:
This comment really just repeats what the puts says below, and so I think it's rather redundant.